core/src/main/java/jenkins/security/LastGrantedAuthoritiesProperty.java

```diff
@@ -136,28 +136,6 @@ protected void failedToLogIn(@NonNull String username) {
             // while this initially seemed like a good idea to avoid allowing wrong impersonation for too long,
             // doing this means a malicious user can break the impersonation capability
             // just by failing to login. See ApiTokenFilter that does the following, which seems better:
```

Comment:
That comment now makes no sense.

Reply:
Since the method is empty and it's base class provides an empty implementation already, is it safe to remove here?

Reply:
Alternatively revert the change; the commented out code looks to me to be more of a documentation nature at the moment, answering the question why there is nothing here.

Reply:
Done.